plugins/python/src/main/kotlin/io/paddle/plugin/python/dependencies/TempVenvManager.kt

```diff
@@ -98,4 +98,18 @@ class TempVenvManager private constructor(val venv: VenvDir, val project: Paddle
     fun getFilesRelatedToPackage(pkg: IResolvedPyPackage): List<File> {
         return InstalledPackageInfoDir.findByNameAndVersion(venv.sitePackages, pkg.name, pkg.version).files
     }
+    private data class PipInstallArgs(val noDeps: Boolean = true, val noCacheDir: Boolean = false) {
+        private val _args = mutableListOf("-m", "pip", "install")
```

Comment:
Could you make vararg parameter out of this field and pass packages at the moment of init? There is only one use case now, and this class is just a DTO, I don't think you really need a Builder pattern here later or smth.